pkgs/by-name/li/limine/package.nix

```diff
@@ -0,0 +1,60 @@
+# Builds limine with all available features.
+
+{
+  # Helpers
+  clangStdenv
+, fetchurl
+, lib
+, # Dependencies
+  llvmPackages
+, mtools
+, nasm
+}:
+
+let
+  version = "7.1.0";
+in
+clangStdenv.mkDerivation {
+  inherit version;
+  pname = "limine";
+  # We don't use the Git source but the release tarball, as the source has a
+  # `./bootstrap` script performing non-standard network access to download
+  # resources (a custom Git submodule-like replacement). Packaging this in Nix
+  # is very cumbersome compared to using the release tarball, where the
+  # `./bootstrap` step was already executed and we can `./configure` right
+  # away.
```

Comment:
My personal opinion is that this comment is overly noisy... The standard way to build autotools software as an end user is to fetch the release tarball... this is not unusual. Just like it is not unusual for `./bootstrap` or `./autogen.sh` to perform extra duties that may involve network access.

Reply:
Thanks! I changed the doc. Hope the new version is better